OpenKeychain/src/main/java/org/sufficientlysecure/keychain/pgp/UncachedKeyRing.java

```diff
@@ -157,6 +159,50 @@ public static UncachedKeyRing decodeFromData(byte[] data)
 
     }
 
+    public static UncachedKeyRing decodeFromData(
```

Comment:
name could be more descriptive, i.e. `decodeFromDataFindExpected` or something like that